test/e2e/page-objects/pages/settings/preinstalled-example-settings.ts

```diff
@@ -57,23 +57,6 @@ class PreinstalledExampleSettings {
     console.log('Checking if the toggle is on');
     await this.driver.waitForSelector(`${this.toggleButton}--on`);
   }
-
-  async checkSelectedRadioOption(option: string): Promise<boolean> {
-    console.log(`Checking if the radio option "${option}" is selected`);
-    const radioOption = await this.driver.findElement(
-      `input[type="radio"][id="${option}"]`,
-    );
-    const isChecked = (await radioOption.getAttribute('checked')) === 'true';
-    return isChecked;
-  }
-
-  async checkSelectedDropdownOption(option: string): Promise<void> {
```

Comment:
I removed this function because it is not effective for checking which dropdown option is selected. Whether `option 1` or `option 2` is selected, the selector we wait for always present. I removed this function to avoid future confusion. See recording:


https://github.com/user-attachments/assets/5ede0c3e-e35a-4d86-9738-10e576e51f8a